annotationsx/settings/aws.py

```diff
@@ -168,6 +173,10 @@
 
 # should be changed depending on platform roles, these are set up for edX
 ADMIN_ROLES = ['urn:lti:instrole:ims/lis/Administrator', 'urn:lti:role:ims/lis/Instructor', 'urn:lti:role:ims/lis/TeachingAssistant']
+
+#TODO: check if this is what we want
+ADMIN_ROLES += ['Administrator', 'Instructor', 'TeachingAssistant']
+
```

Comment:
Some weird bugs were popping up with the previous implementation (for example, we are Instructors but not admins, so we couldn't make courses). If you look at the screenshot below, it appears that there are two fields passed in through the LTI launch: 'ext_roles', and 'roles.' While not completely certain, I believe that the ext_roles contains the type of role you were describing. However, it contains all of them, irrespective of the particular course being accessed.

It seems that 'roles' only sends over the roles of the user in that particular context - so while I'm a student in some other classes, it only sends over the string 

```'Instructor' ```

in this launch, whereas it sends over 

```'urn:lti:instrole:ims/lis/Instructor,urn:lti:instrole:ims/lis/Student,urn:lti:role:ims/lis/Instructor,urn:lti:sysrole:ims/lis/User' ```

for ext_roles.

![screen shot 2015-07-20 at 4 44 43 pm](https://cloud.githubusercontent.com/assets/7493153/8786848/b0a30a06-2efe-11e5-8d95-a940d0e7fc69.png)

![screen shot 2015-07-20 at 4 44 57 pm](https://cloud.githubusercontent.com/assets/7493153/8786851/b29ecc00-2efe-11e5-866d-d8d304a7264d.png)


I'm not completely certain whether the functionality for roles I've described is completely accurate, and I'd like to test it with a few real users to make sure.